docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.7'
+
+services:
+  db:
+    build:
+      context: .
+      dockerfile: docker/db.Dockerfile
+    volumes:
+      - ./volumes:/var/lib/postgresql/data/
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=sso
+    profiles: ["dev", "prod"]
+
+  backend:
+    build:
+      context: .
+      dockerfile: docker/backend.Dockerfile
+    command: sh -c "./cli/create_and_populate_database.bash && python manage.py runserver 0.0.0.0:8100"
+    volumes:
+      - .:/code
+    ports:
+      - "8100:8100"
+    links:
+      - db:db
+    restart: always
+    environment:
+      - DATABASE_HOST=db
+    depends_on:
+      - db
+    profiles: ["dev", "prod"]
+
+  frontend-dev:
+    build:
+      context: frontend
+      dockerfile: Dockerfile
+    command: npm start
+    volumes:
+      - './frontend:/app'
+      - '/app/node_modules'
+    ports:
+      - "3100:3000"
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    profiles: ["dev"]
+
+  frontend-prod:
+    build:
+      context: frontend
+      dockerfile: prod.Dockerfile
+    ports:
+      - '3100:80'
```

Comment:
I'm not sure if we should use port `80` here for now. There is a good chance one might use nginx as a proxy on top of the whole app. Let's use port 3000 as for frontend-dev for the moment